jdbc/pom.xml

```diff
@@ -32,6 +32,201 @@
   <packaging>jar</packaging>
   <version>0.8.0-SNAPSHOT</version>
   <name>Zeppelin: JDBC interpreter</name>
+
+  <profiles>
+    <profile>
+      <id>hive</id>
+      <properties>
```

Comment:
hive jdbc interpreter can support hive1 and hive2 in one interpreter instance ?

Reply:
Haven't seen any error while using jdbc-hive-1.2.1, guess they are backward compatible.